app/src/main/java/dev/beefers/vendetta/manager/network/service/RestService.kt

```diff
@@ -16,7 +16,7 @@ class RestService(
 
     suspend fun getLatestRelease(repo: String) = withContext(Dispatchers.IO) {
         httpService.request<Release> {
-            url("https://api.github.com/repos/vendetta-mod/$repo/releases/latest")
+            url("https://api.github.com/repos/$repo/releases/latest")
```

Comment:
This change is semantically correct. As you can see GitHub themselves refers the org/name segments as "repos" denoted by the segment before the repo.